src/AspNet.Security.OAuth.Zoom/ZoomAuthenticationOptions.cs

```diff
@@ -0,0 +1,37 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using System.Security.Claims;
+using static AspNet.Security.OAuth.Zoom.ZoomAuthenticationConstants;
+
+namespace AspNet.Security.OAuth.Zoom;
+
+/// <summary>
+/// Defines a set of options used by <see cref="ZoomAuthenticationHandler"/>.
+/// </summary>
+public class ZoomAuthenticationOptions : OAuthOptions
+{
+    public ZoomAuthenticationOptions()
```

Comment:
[According to the documentation](https://developers.zoom.us/docs/integrations/oauth/#using-a-client-app-with-pkce), Zoom supports PKCE so [`UsePkce`](https://github.com/dotnet/aspnetcore/blob/c7c76d7d3bb30e29e8386d1a3e6dae9e307fd0de/src/Security/Authentication/OAuth/src/OAuthOptions.cs#L101-L105) should be set to `true` by default.